scripts/qiita-cron-job

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from qiita_db.util import purge_filepaths, empty_trash_upload_folder
+from qiita_db.meta_util import update_redis_stats
+
+
+# This script will perform these jobs:
+# 1. purge_filepaths: remove files from that are leftover in the
+#    qiita.filepath and are present in the filesystem
+# 2. empty_trash_upload_folder: remove files that are present in the trash
+#    of the upload folders
+# 3. update_redis_stats: updates the redis stats information
+#
+# Note that is responsability of the Qiita install system admin to add to a
+# cron job this script and responsible to define how often it should run
+
+
+def call_purge_filepaths():
```

Comment:
Why do you have this intermediate functions? You can call them directly under main.